src/values/expressions/index.js

```diff
@@ -80,7 +78,7 @@ export default function extract(value) {
   }
 
   if (TYPES[type] === undefined) {
-    throw new Error(errorMessage(type));
```

Comment:
maybe instead of throwing tho, it could console.warn? Then we'd still create the possibility of people filing helpful issues, without breaking builds.

Reply:
@ljharb I tried this, but ESLint complains about the `console` statement. I could either suppress the warning or use Node's `Console` object directly. Are you cool with a suppression statement?

Reply:
oh definitely, no-console is pretty much just a rule for code sent to browsers; you could disable it project-wide if you want.